pytorch_starting_to_code.py

```diff
@@ -1,12 +1,20 @@
 # TODO 2024-03: we are setting up a basic working sample of pytorch
 # https://pytorch.org/tutorials/beginner/basics/quickstart_tutorial.html
 
+
```

Comment:
Why is this necessary?

Reply:
Because according to our coding standards at the top of every file we should have a header section.
https://google.github.io/styleguide/javaguide.html